content/patterns/button/examples/button.html

```diff
@@ -127,43 +128,43 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               <td><code>div</code>, <code>a</code></td>
               <td>
                 <ul>
-                  <li>Identifies the element as a <code>button</code> widget.</li>
-                  <li>Accessible name for the button is defined by the text content of the element.</li>
+                  <li>要素を <code>button</code> ウィジェットとして識別します。</li>
+                  <li>ボタンのアクセシブルネームは、要素のテキストコンテンツによって定義されます。</li>
```

Comment:
accessible name を「アクセシブルな名前」としてみました。

```suggestion
                  <li>ボタンの、アクセシブルな名前は、要素のテキストコンテンツによって定義されます。</li>
```